### PR TITLE
docs: add example app for new landing page

### PR DESCRIPTION
**Pull request by bob-watson**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The aio site doesn't have a simple Angular app to use as a demonstration of Angular.

Issue Number: N/A

## What is the new behavior?
The aio site has a simple Angular app to use as a demonstration of Angular.

This PR Is a prerequisite for #47047, which uses this example.
This example needs to be merged so the updates and links in #47047 can be tested before it goes live.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
N/A

### CODE REVIEW

**Review by mgechev — APPROVED** [review]
LGTM!

**Review by mgechev — APPROVED** [review]

File: aio/content/examples/marketing-first-app/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "typeRoots": [
+      "node_modules/@types"
+    ],
+    "lib": [
+      "es2018",
+      "dom"
+    ]
+  },
+  "angularCompilerOptions": {
+    "enableIvy": true,
```
Comment (dgp1130) [inline aio/content/examples/marketing-first-app/tsconfig.json:23]: I don't think we need this option anymore. How did you generate this? Wondering if we need to update the schematic somewhere.

File: aio/content/examples/marketing-first-app/src/index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+    <title>Your first Angular app</title>
+  </head>
+  <body>
+  <add-one-button>Loading...</add-one-button>
```
Comment (dgp1130) [inline aio/content/examples/marketing-first-app/src/index.html:6]: Nit: Indent one level.

**Review by dgp1130 — COMMENTED** [review]
Latest `ng new` should drop `karma.conf.js` and `polyfills.ts`, that would make the example a bit simpler.

**Comment by AndrewKushnir** [comment]
@bob-watson FYI, I'm adding the "cleanup" label for now, since there is a feedback from Doug. Please change the labels back to the merge-ready state once the feedback is addressed.

File: aio/content/examples/marketing-first-app/tsconfig.json (hunk 1 above)
Comment (bob-watson) [reply aio/content/examples/marketing-first-app/tsconfig.json:23]: @dgp1130 Thanks, I think it was created earlier in the summer/fall, so it probably used a v14 schematic.

File: aio/content/examples/marketing-first-app/src/index.html (hunk 2 above)
Comment (bob-watson) [reply aio/content/examples/marketing-first-app/src/index.html]: Fixed in e9436d7.

**Comment by bob-watson** [comment]
I'm not sure what's causing the problems in the [ci/circleci: legacy-unit-tests-saucelabs](https://circleci.com/gh/angular/angular/1266948) test. @AndrewKushnir  any ideas?

**Comment by AndrewKushnir** [comment]
@bob-watson it turned out to be a flake, I've restarted that CI job and it came back "green".

**Comment by bob-watson** [comment]
@dgp1130 I couldn't remove the files without causing problems. I addressed all the other feedback.
I'd like to get this much merged into main so I can fix up the change in #47047 to test the links on our new marketing pages. I can re-create the app with the ng v15 CLI in the #47047, if this they still need to be removed.

**Review by dgp1130 — APPROVED** [review]
@bob-watson, we don't need to block on an updated CLI project. I would definitely like to see a follow up which removes those files though. Part of the reasoning behind hiding them is to make documentation like this more approachable so I think it's important to do here when we can.

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 65d8803f1103c631450537cbd3d9dfbe06661ff3.

**Pull request closed without merging**